modules/ascat/main.nf

```diff
@@ -2,7 +2,7 @@ process ASCAT {
     tag "$meta.id"
     label 'process_medium'
 
-    conda (params.enable_conda ? "bioconda::ascat=3.0.0-1 bioconda::cancerit-allelecount=4.3.0-1": null)
+    conda (params.enable_conda ? "bioconda::ascat=3.0.0 bioconda::cancerit-allelecount=4.3.0-2": null)
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
         'https://depot.galaxyproject.org/singularity/mulled-v2-c278c7398beb73294d78639a864352abef2931ce:dfe5aaa885de434adb2b490b68972c5840c6d761-0':
```

Comment:
Does the container have the same version? Should also prob be updated in the version below

Reply:
No worries - I'll use the same version for sure; I'm just testing why it fails linting. Seems it can't really find that version and I thought it could be something to do with the -1 -2, but seems like it's not

Reply:
ok I think I am missing something, the latest release is 3.0.0, no? https://github.com/VanLoo-lab/ascat/releases/tag/v3.0.0

Reply:
Yes, the latest release of ASCAT is 3.0.0 and that's the one it's using. But ASCAT 3.0.0 must have allelecount-4.3.0 available in path to run. So it's a mulled container with the two.

Reply:
but conda 3.0.0-1?

Reply:
🤦‍♀️ sorry ignore me, you are removing the -1. need more coffee ☕